javascript/package.json

```diff
@@ -64,7 +64,12 @@
     "@tensorflow/tfjs-vis": "^1.5.1"
   },
   "dependencies": {
-    "@tensorflow/tfjs": "^4.2.0",
-    "@tensorflow/tfjs-vis": "^1.5.1"
+    "@jridgewell/sourcemap-codec": "^1.4.14",
+    "@types/uuid": "^9.0.1",
+    "aws-sdk": "^2.1346.0",
+    "axios": "^1.3.4",
```

Comment:
The 3rd party dependencies should be installed as `peerDependencies` and externalize those from final bundle.

- [externalize-frameworks](https://github.com/frehner/modern-guide-to-packaging-js-library?s=09#externalize-frameworks)